docs/howto/test.md

```diff
@@ -105,6 +105,30 @@ default schema path in an initializer.
 ActiveModelSerializers.config.schema_path = 'spec/support/schemas'
 ```
 
+### RSpec test helpers
+
+ActiveModelSerializers provides a `have_valid_schema` matcher to be used on your controller/request specs to
+validate the response against a [JSON Schema](http://json-schema.org/). Let's look at the equivalent setup for the example above.
+
+```ruby
+# rails_helper.rb
+require 'active_model_serializers/rspec'
+
+# spec/controller/posts_controller_spec.rb
+describe PostsController do
```

Comment:
But actually, we should give an example from a request spec, since controller specs aren't very useful and are going away. Some discussion in https://github.com/rails-api/active_model_serializers/issues/1970#issuecomment-262642416

```ruby
RSpec.describe PostsController, type: :request do


end
```